store-in-gcs/fake-event.py

```
@@ -0,0 +1,296 @@
+#!/usr/bin/env python3
+"""
+Eventarc Firestore Curl Command Generator
+Generates curl commands for google.cloud.firestore.document.v1.written events
+"""
+
+import json
+import uuid
+import base64
+import argparse
+from datetime import datetime, timezone
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)